app/(main)/store/order-list/[id]/_components/order-list/OrderList.tsx

```diff
@@ -0,0 +1,45 @@
+'use client';
+
+import notification from '../../../../../../_assets/images/notification.png';
+import { useState } from 'react';
+import OrderListDropDown from '../order-list-dropdown/OrderListDropDown';
+import fetchOrderList from '../../fetchOrderList';
+import OrderListModal from '../order-list-modal/OrderListModal';
+import Image from 'next/image';
+import InfiniteScrollList from '@/app/_components/infinite-scroll/InfiniteScrollList';
+import OrderListCard from '../order-list-card/OrderListCard';
+import Footer from '@/app/_components/Footer/Footer';
+
+const OrderList = () => {
+  const [isOpen, setIsOpen] = useState(false);
+
+  const onClickOrderList = () => {
+    setIsOpen(() => !isOpen);
```

Comment:
`prev`이름은 달라도 상관없지만 이렇게 쓰는게 더 좋아 보여요..!

```suggestion
    setIsOpen((prev) => !prev);
```